packages/block-library/src/section/edit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	InspectorControls,
+	InnerBlocks,
+	PanelColorSettings,
+	withColors,
+} from '@wordpress/block-editor';
+
+function SectionEdit( { className, setBackgroundColor, backgroundColor } ) {
+	const styles = {
+		backgroundColor: backgroundColor.color,
+	};
+
+	const classes = classnames( className, backgroundColor.class, {
+		'has-background': !! backgroundColor.color,
+	} );
+
+	return (
+		<Fragment>
+			<InspectorControls>
+				<PanelColorSettings
+					title={ __( 'Color Settings' ) }
+					colorSettings={ [
+						{
+							value: backgroundColor.color,
+							onChange: setBackgroundColor,
+							label: __( 'Background Color' ),
+						},
+					] }
+				/>
+			</InspectorControls>
+			<div className={ classes } style={ styles }>
```

Comment:
👋. I know this PR is already merged but I'm wondering if such a generic `div` element can be extracted in its own component. It will probably be an opportunity to make the Section block more cross-platform ready by not using html tags directly. WDYT @getdave ?

cc @gziolo , @Tug , @koke

Reply:
@hypest, we totally should introduce `View` component which will work cross-platform. @koke landed a similar component yesterday. See https://github.com/WordPress/gutenberg/pull/14361 for reference. It should be a very similar use case.